tests/http/test_request_normalization.py

```diff
@@ -2,11 +2,12 @@
 import urllib
 
 import six
+from six.moves import urllib_parse as urlparse
 
 import requests
 
 from flex.http import (
-    normalize_request, _tornado_available, _falcon_available, _webob_available
+    normalize_request, _tornado_available, _falcon_available, _webob_available, _django_available
```

Comment:
would you mind splitting these imports to one per line now that they are getting long